.github/workflows/kind-e2e.yaml

```diff
@@ -64,13 +64,11 @@ jobs:
         # Install Eventing Core.
         kubectl apply --filename https://storage.googleapis.com/knative-nightly/eventing/latest/eventing-crds.yaml
         sleep 5
-        kubectl apply --filename https://storage.googleapis.com/knative-nightly/eventing/latest/eventing-core.yaml
 
-        # Use the In-Memory Channel
-        kubectl apply --filename https://storage.googleapis.com/knative-nightly/eventing/latest/in-memory-channel.yaml
+        kubectl apply --filename https://storage.googleapis.com/knative-nightly/eventing/latest/eventing-core.yaml \
```

Comment:
this is no longer necessary?

Reply:
It is at this point in the sequence.   My third change removes the sinkbinding dep, and then this can go...